db/migrations/20240613150913_category.up.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
+CREATE TABLE IF NOT EXISTS Category (
+    markdown_id UUID PRIMARY KEY REFERENCES Markdown(markdown_id),
+    category TEXT,
+    embedding DOUBLE PRECISION[]
```

Comment:
added `embedding` column. It stores the array of floats

Reply:
I _LOVE_ that you figured this out on your own nicely done!!

We might eventually use a Postgres plugin and something slightly different here, but a problem for another day!
This is perfect!